doc/sphinx/proof-engine/ltac.rst

```diff
@@ -532,42 +635,47 @@ tactic that succeeds from a list of tactics:
 
             Tactic Notation "foo" tactic_list(tacs) := first tacs.
 
-Left-biased branching: ||
-~~~~~~~~~~~~~~~~~~~~~~~~~
+First tactic to make progress: ||
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Yet another way of branching without backtracking is the following
 structure:
 
-.. tacn:: @ltac_expr__1 || @ltac_expr__2
-   :name: || (left-biased branching)
+.. tacn:: @ltac_expr1 || {| @ltac_expr2 | @binder_tactic }
+   :name: || (first tactic making progress)
 
-   :n:`@ltac_expr__1` and :n:`@ltac_expr__2` are evaluated to :n:`v__1` and
-   :n:`v__2` respectively, which must be tactic values. The tactic value :n:`v__1` is
-   applied in each subgoal independently and if it fails *to progress* then
-   :n:`v__2` is applied. :n:`@ltac_expr__1 || @ltac_expr__2` is
-   equivalent to :n:`first [ progress @ltac_expr__1 | @ltac_expr__2 ]` (except that
-   if it fails, it fails like :n:`v__2`). Branching is left-associative.
+   :n:`@ltac_expr1 || @ltac_expr2` is
+   equivalent to :n:`first [ progress @ltac_expr1 | @ltac_expr2 ]`, except that
+   if it fails, it fails like :n:`v__2`. Branching is left-associative.
```

Comment:
```suggestion
   if it fails, it fails like :n:`ltac_expr__2`. Branching is left-associative.
```
Also, is it clear that "branching" means `||`?

Reply:
Replaced "Branching" with ||.